packages/nx/src/adapter/ngcli-adapter.ts

```diff
@@ -1209,18 +1214,19 @@ async function getWrappedWorkspaceNodeModulesArchitectHost(
         join(dirname(packageJsonPath), executorsFile)
       );
       const executorsJson = readJsonFile<ExecutorsJson>(executorsFilePath);
-      const executorConfig: {
-        implementation: string;
-        batchImplementation?: string;
-        schema: string;
-        hasher?: string;
-      } =
+      const executorConfig =
         executorsJson.builders?.[builder] ?? executorsJson.executors?.[builder];
       if (!executorConfig) {
         throw new Error(
           `Cannot find builder '${builder}' in ${executorsFilePath}.`
         );
       }
+      if (typeof executorConfig === 'string') {
+        // Angular CLI can have a builder pointing to another package:builder
+        const [packageName, executorName] = executorConfig.split(':');
+        return this.readExecutorsJson(packageName, executorName);
+      }
```

Comment:
Interesting, this is something like this right?:

```json
{
  "builders": {
    "build": "@company/other-pkg:build"
  }
}
```

Reply:
Yes. See https://github.com/angular/angular-cli/blob/main/packages/angular_devkit/build_angular/builders.json#L4.